04-materials/02-anatomy-of-extensions.md

```diff
@@ -99,24 +99,54 @@ First we'll look at the final extension together. It:
 
 ### Create a repository in GitHub and clone it
 
+If you're an experienced Git & GitHub user, feel free to do this step the way you
+normally would!
+
 0. Change to the parent directory where you want to work, e.g.
 
-    ```bash
-    cd ~/Projects
-    ```
+   ```bash
+   cd ~/Projects
+   ```
 
-1. Create a repository in GitHub and clone it
+1. If you don't already have GitHub authentication set up on your local machine,
+   authenticate with GitHub.
 
-    ```bash
-    # TODO: Test
-    gh repo create jupytercon2025-extension-workshop --public --clone
-    ```
+   ```bash
+   gh auth login
+   ```
 
-2. Change directory into your new repository
+   Select: `GitHub.com`, `HTTPS`, `Yes`, and `Login with a web browser`, then follow the
+   instructions carefully.
+   Then, set up the Git CLI to authenticate with GitHub:
 
-    ```bash
-    cd jupytercon2025-extension-workshop
-    ```
+   ```bash
+   gh auth setup-git
+   ```
+
+2. Create a repository in GitHub and clone it:
+
+   ```bash
+   gh repo create jupytercon2025-extension-workshop --public --clone
+   ```
```

Comment:
Would you like to ask them to set the repo topics and description for better discovery
```
gh repo edit my-new-repo --add-topic "jupyterlab-extension" --add-topic "jupytercon2025" 
```